tests/integration/llm/deepspeed-model.py

```diff
@@ -15,7 +15,16 @@ def load_model(properties):
     if "model_id" in properties:
         model_location = properties['model_id']
     logging.info(f"Loading model in {model_location}")
-    model = AutoModelForCausalLM.from_pretrained(model_location,
+    checkpoint = None
+    if "checkpoint" in properties:
+        checkpoint = os.path.join(model_location, properties['checkpoint'])
+
+    if checkpoint:
+        config_file = os.path.join(model_location, "config.json")
+        config = AutoConfig.from_pretrained(config_file)
+        model = AutoModelForCausalLM.from_config(config)
```

Comment:
You may also want to use meta tensor loading?

Reply:
Added it and added checkpoint and base_dir to deepspeed.init_inference